src/PropTypesCompletionItemProvider.ts

```diff
@@ -47,6 +49,22 @@ export default class PropTypesCompletionItemProvider implements CompletionItemPr
         return documentText.slice(startTagPosition, endTagPosition + 1).replace(/\s{2,}/g, ' ');
     }
 
+    private getPropTypesFromJsxTag(jsxTag: string): string[] {
+        /* 
+            this method may return not only proptypes
+            but also some other keywords that any jsx tag tag may contain
+        */
+        const regex = /[^\s]*=/g;
+        let parsedProptypesWithEqualsSigns: RegExpMatchArray | null = jsxTag.match(regex);
```

Comment:
`let parsedProptypesWithEqualsSigns: RegExpMatchArray = jsxTag.match(regex) || [];`

With this code you can remove "if" above and "!" in 62 line because anyway you will have valid value in "parsedProptypesWithEqualsSigns"